libs/common-types/src/lib/con-profile/con-profile.mapper.ts

```diff
@@ -113,6 +113,7 @@ export class ConProfileMapper
     props.Profile_Status__c = srcProps.profileStatus
     // props.RecordType.DeveloperName = srcProps.RecordType.userType
     props.ReDI_Location__c = srcProps.rediLocation
+    props.Partnership_Mentor__c = srcProps.mentor_isPartnershipMentor
```

Comment:
Field should be added the "other direction" as well, i.e. to `fromPersistence` function further up

Reply:
I didn't get your comment @ericbolikowski

Reply:
The job of a mapper is to convert objects from persistence model to domain model. And vice versa.

At the moment the mapper is only mapping one way - from domain model to persistence model, not the other way. The result will be that only changes to `mentor_isPartnershipMentor` will be persisted. But the value of it will never be correctly read, since we don't include that when we map from persistence model to domain model.

That's why I say we need to add the "other direction" to the other function. Scroll up and you'll see lots of mappings in the other direction.

Reply:
@helloanil, is this resolved?

Reply:
I believe it is now :)